src/cdk/tree/tree.ts

```diff
@@ -159,6 +290,35 @@ export class CdkTree<T, K = T> implements AfterContentChecked, CollectionViewer,
     }
   }
 
+  ngAfterContentInit() {
```

Comment:
One practice I push for is that component lifecycle method like this should generally be pretty light, with all of the behavior living in other well-named functions. ([example](https://github.com/angular/components/blob/a2ebcbc0e582124a81b89687eb519c075ceb69f0/src/cdk-experimental/selection/select-all.ts#L81))

Reply:
done